src/test/browser/ETH2GNO.test.ts

```diff
@@ -489,4 +495,26 @@ describe('ETH 2 GNO contract', () => {
     console.log(`= ${totalETH}\t${totalGNO}`)
   }
 
+  function watchAllEventsFor(contract: any, name: string) {
+    const addr2acc = Object.entries(accs).reduce((accum, [name, addr]) => (accum[addr] = name, accum), {})
```

Comment:
Not sure how you feel about this but let me know either way, would love to see an example of what these reduce fns spit out, sometimes when going through reviews its a bit tricky here

Reply:
Given `accs = {master: '0x23df...'}`  it flips `key/val` transforming into `addr2acc = { '0x23df...': "master"}`

Reply:
`addr 2 acc` makes sense 😅